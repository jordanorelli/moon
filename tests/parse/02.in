a_string: "a string value"

an_int: 1
another_int: +9
moar_int: -12

a_float: 1.0
another_float: -.9
extra_floaty: +1.2

complex1: 1+1i
complex2: 1+0i
complex3: 1.3+4.7i
complex4: -4.2+8.9i
complex5: +4.2-8.9i


