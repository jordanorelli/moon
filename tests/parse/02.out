root:
  assign:
    name:
      a_string
    value:
      string:
        a string value
  assign:
    name:
      an_int
    value:
      int:
        1
  assign:
    name:
      another_int
    value:
      int:
        9
  assign:
    name:
      moar_int
    value:
      int:
        -12
  assign:
    name:
      a_float
    value:
      float:
        1
  assign:
    name:
      another_float
    value:
      float:
        -0.9
  assign:
    name:
      extra_floaty
    value:
      float:
        1.2
  assign:
    name:
      complex1
    value:
      complex:
        (1+1i)
  assign:
    name:
      complex2
    value:
      complex:
        (1+0i)
  assign:
    name:
      complex3
    value:
      complex:
        (1.3+4.7i)
  assign:
    name:
      complex4
    value:
      complex:
        (-4.2+8.9i)
  assign:
    name:
      complex5
    value:
      complex:
        (4.2-8.9i)
